src/odemis/driver/hamamatsurx.py

```diff
@@ -0,0 +1,2040 @@
+                    ShowSpectrControl: Shows or hides the Spectrograph status/control dialog
+        :returns: Label, Current value (bool), Param Type (PARAM_TYPE_BOOL)"""
+        # TODO check with HW
+        try:
+            label, val, typ = self.sendCommand("GenParamInfo", parameter)
+            param_typ = int(typ)
+            value = bool(val)
+        except (IndexError, TypeError, ValueError) as ex:
+            raise IOError("Failed to decode response from GenParamInfo: %s" % ex)
+        return label, value, param_typ
+
+    def GenParamInfoEx(self, parameter):
+        """Returns the information about the specified parameter. Returns more detailed information
+        in case of a list parameter (Parameter type = 2) than GenParamInfo.
+        :parameter parameter: (str) see GenParamInfo
+        :returns: Label, Current value (bool), Param Type (PARAM_TYPE_BOOL)"""
+        # TODO GenParamInfoEx should return more info than GenParamInfo,
+        # but the docs was probably not more detailed regarding this. Code might fail...
+        # TODO check this with HW!
+        try:
+            label, val, typ = self.sendCommand("GenParamInfoEx", parameter)
+            param_typ = int(typ)
+            value = bool(val)
+        except (IndexError, TypeError, ValueError) as ex:
+            raise IOError("Failed to decode response from GenParamInfo: %s" % ex)
+        return label, value, param_typ
+
+    def GenParamsList(self):
+        """Returns a list of all parameters related to the general options.
+        :returns: NumberOfParameters,Parameter1,..., ParameterN."""
+        return self.sendCommand("GenParamsList")
+
+    # === Acquisition commands ========================================================
+
+    def AcqStart(self, AcqMode):
+        """Start an acquisition.
+        :parameter AcqMode: (str) Live: Live mode
+                      SingleLive: Live mode (single exposure)
+                      Acquire: Acquire mode
+                      AI: Analog integration
+                      PC: Photon counting"""
+        if not self._readoutcam.t_image.isAlive():  # restart thread in case it was terminated
+            self.AcqLiveMonitor("RingBuffer", nbBuffers=10)
+            self._readoutcam.t_image = threading.Thread(target=self._readoutcam._getDataFromBuffer)
+            self._readoutcam.t_image.start()
+        self.sendCommand("AcqStart", AcqMode)
+
+    def AcqStatus(self):
+        """Returns the status of an acquisition.
+        :return: status, mode"""
+        return self.sendCommand("AcqStatus")
+
+    def AcqStop(self, timeout=1):
+        """Stops the currently running acquisition.
+        :parameter timeout: (0.001<= float <=60) The timeout value (in s)
+        until this command should wait for an acquisition to end.
+        :return: (float) timeout (in s)"""
+        # Note: RemoteEx needs timeout in ms
+        timeout = timeout * 0.001
```

Comment:
To convert from s to ms, you need to multiply by 1000.
=> str(timeout * 1000)
For the return value, I'd assume that sendCommand returns a `list`. So better write:
```
ans = self.sendCommand("AcqStop", str(timeout * 1000))
rtimeout = float(ans[0]) * 0.001
return rtimeout
```